config/sidekiq_scheduler.yml

```diff
@@ -166,6 +166,11 @@ VRE::CreateCh31SubmissionsReportJob:
   include_metadata: true
   description: "Send a daily report to the VRE team about Chapter 31 submissions"
 
+Mobile::WeeklyMaintenanceWindowLogger:
+  cron: "0 13 * * 1 America/New_York"
```

Comment:
I generally prefer to run crons late at night because the servers are idle. Probably not a big deal in this case because the job isn't very resource intensive.

Reply:
I figure the way things work around here is late at night is when upstream dependencies go down for maintenance so wanted to avoid the job failing due to this.

Reply:
Also quite possible. Probably no need to change this.